rotation.go

```diff
@@ -384,6 +386,17 @@ func (b *backend) setStaticAccountPassword(ctx context.Context, s logical.Storag
 		err = b.client.UpdateUserPassword(config.LDAP, input.Role.StaticAccount.Username, newPassword)
 	}
 	if err != nil {
+		if usedExistingCredential {
+			// A retried password has failed again. Delete WAL and try with fresh password
```

Comment:
nit: This comment is a little unclear to me. I have context here so I understand what it is trying to say. But I am not so sure about someone reading this that doesn't have the context.

I think if we renamed `usedExistingCredential` to `usedCredentialFromPreviousRotation` then we could omit this comment entirely. If we do that, I think the comment on line 396 should be sufficient to explain the expectations going forward.

Reply:
Updated in latest, thanks!